features/google/rollup.config.mjs

```diff
@@ -7,7 +7,7 @@ import serve from "rollup-plugin-serve";
 import svg from "rollup-plugin-svg";
 import replace from "@rollup/plugin-replace";
 import genPodjs from "@polypoly-eu/podjs/rollup-plugin-gen-podjs/genPodjs.js";
-import sillyI18n from "@polypoly-eu/silly-i18n/rollup-plugin.cjs";
+import sillyI18n from "@polypoly-eu/silly-i18n/rollup-plugin.js";
```

Comment:
In this case you can probably simply eliminate the extension, but that's not important.

Reply:
I left the extension since omitting the extension when importing a module using ESM syntax in non-standard. Then again package imports are non-standard anyway. But other bundlers (e.g. webpack in strict ESM mode) might fail.

Reply:
Just in case, we're not likely to ever use webpack.